gazelle_haskell_modules/rule_generation.go

```diff
@@ -119,15 +119,19 @@ func haskellModuleRulesToRuleInfos(
 		}
 
 		modDatas := haskellModulesToModuleData([]string{src})
-		ruleInfo := RuleInfo{
-			OriginatingRules: oRules,
-			ModuleData:       modDatas[0],
-		}
+		if len(modDatas) == 1 {
```

Comment:
I implemented this in b2f66f2

Reply:
Thanks. I wonder if we should (in another issue perhaps) change the logic here to error if we receive `len(modData) > 1`, as an assert, since it shouldn't be possible?

Reply:
I'd be fine with such a change.

Reply:
I'll open an issue after we merge this PR.

Reply:
#28